src/grammar.pest

```diff
@@ -58,8 +58,8 @@
         comment = _{ "#" ~ ( !NEWLINE ~ !eoi ~ ANY )* }
 
         atomic = _{ prefixed | standalone }
-            prefixed = { prefix* ~ postfixed }
-            postfixed = { atom ~ postfix* }
+            prefixed = { prefix* ~ WS* ~ postfixed }
+            postfixed = { atom ~ WS_NO_NL* ~ postfix* }
```

Comment:
All expressions are "atomic" (I think I got this name from some examples in `pest`... they're not atomic in the same sense as atomic data - they're more like "pieces") and can be prefixed or postfixed. 

Here I permit any whitespace after a prefix op and any non-newline whitespace before a postfix op. 

Disallowing newlines before a postfix is necessary to prevent 

```
y <- x
(x + 3)
```

from being parsed as

```
y <- x(x + 3)
```